addons/account/models/ir_module.py

```diff
@@ -72,8 +72,8 @@ def try_loading(env):
             self.env.registry._auto_install_template = try_loading
         return res
 
-    def _load_module_terms(self, modules, langs, overwrite=False):
-        super()._load_module_terms(modules, langs, overwrite)
+    def _load_module_terms(self, modules, langs, **kwargs):
```

Comment:
I prefer to define all params explicitly when override, they are not too many and we don't expect anyone extends the api

Reply:
I agree, it's only one more. I also agree that we don't want anyone to extend.
But since keyword arguments were not used here, I defined that way so that we don't have to change this signature anytime the parent signature changes.
I guess both options are fine... I applied yours anyway

Reply:
There is a small difference
when you do `def _load_module_terms(self, modules, langs, **kwargs):`
this method cannot be called like `module._load_module_terms(['module_name'], ['fr_FR'], True)`
so you actually change the api but not only extend it.